### PR TITLE
{bio,lib}[GCCcore/10.2.0] vcflib v1.0.2, fsom v20141119, intervaltree v0.1, ... w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by deniskristak**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/fsom/fsom-20141119-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+github_account = 'ekg'
+source_urls = [GITHUB_SOURCE]
+sources = ['%s.tar.gz' % local_commit]
+patches = [
+    '%(name)s-20141119_build-libs.patch',
+    '%(name)s-20141119_fix-abs-overload.patch'
+]
```
Comment (verdurin) [inline easybuild/easyconfigs/f/fsom/fsom-20141119-GCCcore-10.2.0.eb:19]: @deniskristak please add the missing checksums

**Review by verdurin — DISMISSED** [review]

**Comment by deniskristak** [comment]
@verdurin my comments from the original eb_inuits issue:

_PRs related to this issue:
easybuilders/easybuild-easyconfigs#13472
easybuilders/easybuild-easyconfigs#13473
easybuilders/easybuild-easyconfigs#13474

we also need to tweak two existing easyconfigs (shouldnt affect existing easyconfigs wrt dependencies):
easybuilders/easybuild-easyconfigs#13477

and finally snippy - easybuilders/easybuild-easyconfigs#13475_

so all the PRs are dependent on each other and since you merged one PR, I think you can continue testing (another PR should have all deps it needs right now...)

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 6 out of 6 (4 easyconfigs in total)
node374.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/9f1aa78a71971524135cf48f991acbd2 for a full test report.

**Review by lexming — APPROVED** [review]
@deniskristak thanks for these updates. I have moved some easyconfigs from your other PRs into this one, otherwise it was not possible to merge `vsclib` due to missing dependencies.

LGTM

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 8 out of 8 (7 easyconfigs in total)
node374.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/f6ca4ad82b87bc107f0c03e000527c31 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @deniskristak !

**Pull request merged**